website_portal_medical_prescription_order_line/controllers/main.py

```diff
@@ -28,6 +28,21 @@ def my_medical(self, **kw):
             ('patient_id.parent_id', 'child_of', [partner_id.id]),
         ])
 
+        url = str(request.httprequest)
```

Comment:
And what is the return from `request.args.getlist('patients')`? The behavior you are describing [directly conflicts with the documentation of Werkzeug](http://werkzeug.pocoo.org/docs/0.11/datastructures/#werkzeug.datastructures.MultiDict.getlist), so I am finding it hard to believe it doesn't work.